filament/backend/src/vulkan/VulkanFboCache.h

```diff
@@ -44,14 +44,14 @@ class VulkanFboCache {
         VkFormat depthFormat; // 4 bytes
         union {
             struct {
-                uint8_t clear;
-                uint8_t discardStart;
-                uint8_t discardEnd;
-                uint8_t dependencies;
+                TargetBufferFlags clear : 8;
```

Comment:
isn't TargetBufferFlags a uint8_t? I don't think we need the bitfield here

Reply:
Good catch, fixed.